src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,192 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import * as Sentry from '@sentry/browser';
+
+import Button from 'app/components/button';
+import CommitRow from 'app/components/commitRow';
+import {Panel} from 'app/components/panels';
+import {promptsUpdate} from 'app/actionCreators/prompts';
+import robotLogo from 'app/../images/robot-logo.png';
+import SentryTypes from 'app/sentryTypes';
+import {snoozedDays} from 'app/utils/promptsActivity';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import withApi from 'app/utils/withApi';
+
+const DUMMY_COMMIT = {
+  id: 'ff721e0',
+  author: {name: 'Uh/Oh'},
+  dateCreated: moment()
+    .subtract(3, 'day')
+    .format(),
+  repository: {
+    provider: {id: 'integrations:github', name: 'GitHub', status: 'active'},
+  },
+  message: t('This commit accidentally broke something'),
+};
+
+class EventCauseEmpty extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    project: SentryTypes.Project.isRequired,
+  };
+
+  state = {
+    shouldShow: undefined,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    const {api, project, organization} = this.props;
+
+    api.request('/promptsactivity/', {
```

Comment:
Lets use async await here if we can

```jsx
async fetchData() {
  const {api, project, organization} = this.props;

  const data = await api.requestPromise('/promptsactivity/', {
    query: {
      project_id: project.id,
      organization_id: organization.id,
      feature: 'suspect_commits',
    },
  });
 
  this.setState({shouldShow: this.shouldShow(data)});
}
```

The error will be captured by sentry if it fails.